sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/ScriptTransformation.scala

```diff
@@ -328,23 +361,27 @@ case class HiveScriptIOSchema (
     (columns, columnTypes)
   }
 
-  private def initSerDe(
+  private def createTableProperties(
       serdeClassName: String,
       columns: Seq[String],
       columnTypes: Seq[DataType],
-      serdeProps: Seq[(String, String)]): AbstractSerDe = {
-
-    val serde = Utils.classForName(serdeClassName).newInstance.asInstanceOf[AbstractSerDe]
-
+      serdeProps: Seq[(String, String)]) = {
     val columnTypesNames = columnTypes.map(_.toTypeInfo.getTypeName()).mkString(",")
-
     var propsMap = serdeProps.toMap + (serdeConstants.LIST_COLUMNS -> columns.mkString(","))
     propsMap = propsMap + (serdeConstants.LIST_COLUMN_TYPES -> columnTypesNames)
-
+    propsMap = propsMap + (serdeConstants.FIELD_DELIM -> "\t")
```

Comment:
line delimiter is control by RecorderWriter, see TextRecordWriter as an example:
``` scala
  public void write(Writable row) throws IOException {
    Text text = (Text) row;
    Text escapeText = text;

    if (HiveConf.getBoolVar(conf, HiveConf.ConfVars.HIVESCRIPTESCAPE)) {
      escapeText = HiveUtils.escapeText(text);
    }

    out.write(escapeText.getBytes(), 0, escapeText.getLength());
    out.write(Utilities.newLineCode);
  }
```